experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricData.ts

```diff
@@ -57,6 +57,7 @@ export type MetricData = SingularMetricData | HistogramMetricData;
 export interface InstrumentationLibraryMetrics {
   instrumentationLibrary: InstrumentationLibrary;
   metrics: MetricData[];
+  errors: unknown[];
```

Comment:
Is this a part of the proto definition that I missed? Why use an unknown type?

Reply:
No, this is not part of the proto. I think we can log out these errors in exporters that don't support exporting errors to the backend.

These errors can be thrown by userland code. We can not assume their types so an `unknown` type is the best match.